qiskit/visualization/pulse_v2/style/default.json

```diff
@@ -0,0 +1,86 @@
+{
+  "formatter": {
+    "general": {
+        "fig_size": [8, 6],
+        "dpi": 150
+    },
+    "color": {
+        "fill_waveform_d": ["#648fff", "#002999"],
+        "fill_waveform_u": ["#ffb000", "#994A00"],
+        "fill_waveform_m": ["#dc267f", "#760019"],
+        "fill_waveform_a": ["#dc267f", "#760019"],
+        "baseline": "#000000",
+        "barrier": "#222222",
+        "background": "#f2f3f4",
+        "annotate": "#222222",
+        "frame_change": "#000000",
+        "snapshot": "#000000",
+        "axis_label": "#000000"
+    },
+    "alpha": {
+        "fill_waveform": 1.0,
+        "baseline": 1.0,
+        "barrier": 0.7
+    },
+    "layer": {
+        "fill_waveform": 2,
+        "baseline": 1,
+        "barrier": 1,
+        "annotate": 4,
+        "axis_label": 4,
+        "frame_change": 3,
+        "snapshot": 3
+    },
+    "margin": {
+        "top": 0.2,
+        "bottom": 0.2,
+        "left": 0.05,
+        "right": 0.05,
+        "between_channel": 0.1
+    },
+    "label_offset": {
+        "pulse_name": -0.1,
+        "scale_factor": -0.1,
+        "frame_change": 0.1,
+        "snapshot": 0.1
+    },
+    "text_size": {
+        "axis_label": 15,
+        "annotate": 12,
+        "frame_change": 20,
+        "snapshot": 20,
+        "fig_title": 15
+    },
+    "line_width": {
+        "fill_waveform": 0,
+        "baseline": 1,
+        "barrier": 1
+    },
+    "line_style": {
+        "fill_waveform": "-",
+        "baseline": "-",
+        "barrier": ":"
+    },
+    "control": {
+        "apply_phase_modulation": true,
+        "show_snapshot_channel": false,
+        "show_acquire_channel": false,
+        "show_empty_channel": false
+    },
+    "unicode_symbol": {
+        "frame_change": "\\u21BA",
+        "snapshot": "\\u21AF"
+    },
+    "latex_symbol": {
+        "frame_change": "\\circlearrowleft",
+        "snapshot": "None"
+    }
+  },
+  "generator": {
+    "waveform": "None",
```

Comment:
How would these be handled if functions need to be specified?

Reply:
I don't plan to preset generators because they should be controlled by the stylesheet. The purpose of default style is to avoid writing misc setting multiple times.